tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -124,6 +127,7 @@ describe('Header', () => {
 
     const secondaryLinks = wrapper.find('.usa-nav__secondary-item')
 
-    expect(secondaryLinks.length).toEqual(0)
+    expect(secondaryLinks.first().hasClass('display-none')).toBeTruthy()
+    expect(secondaryLinks.last().text()).toEqual('Sign In')
```

Comment:
Adding tests for header component. Testing the new `Sign In` and `Sign Out` authentication change.